### PR TITLE
Mod/all

### PR DESCRIPTION
**Pull request by ken2403**

### CODE REVIEW

File: pyg_material/data/dataset.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def __init__(
     def len(self) -> int:
```
Comment (ken2403) [inline pyg_material/data/dataset.py:29]: Modify dataset(input: Structure list, output: torch_geometric.data.Dataset)

File: pyg_material/model/base.py [hunk 2]
```diff
@@ -80,3 +80,6 @@ def get_triplets(self, batch: Batch) -> tuple[Tensor, Tensor, Tensor, Tensor, Te
         edge_idx_ji = adj_t_row.storage.row()[mask]
 
         return (idx_i, idx_j, tri_idx_i, tri_idx_j, tri_idx_k, edge_idx_kj, edge_idx_ji)
+
```
Comment (ken2403) [inline pyg_material/model/base.py:83]: add numparam func

File: pyg_material/model/wfnet.py [hunk 3]
```diff
@@ -14,99 +14,109 @@
 from pyg_material.nn import Dense
 from pyg_material.utils import activation_resolver, init_resolver
 
-# 1s, 2s, 2p, 3s, 3p, 4s, 3d, 4p, 5s, 4d, 5p, 6s, 4f, 5d, 6p
+# 1s, 2s, 2p, 3s, 3p, 4s, 3d, 4p, 5s, 4d, 5p, 6s, 4f, 5d, 6p, 7s, 5f, 6d
```
Comment (ken2403) [inline pyg_material/model/wfnet.py:17]: add 2 orbital

File: pyg_material/model/wfnet.py [hunk 4]
```diff
@@ -176,28 +188,17 @@ def __init__(self, cutoff: float | None, standarize: bool = True):
...
-            rbf = func(area)
-        return 1 / (torch.sqrt(torch.sum(rbf**2, dim=0, keepdim=True)) + 1e-12)
 
     def forward(self, dist: Tensor) -> Tensor:
-        device = dist.device
-        if self.stadarize:
-            rbf = torch.stack([f(dist) * n.to(device) for f, n in zip(self.basis_func, self.normalize_coeff)], dim=1)
-        else:
-            rbf = torch.stack([f(dist) for f in self.basis_func], dim=1)
+        rbf = torch.stack([f(dist, self.a0) for f in self.basis_func], dim=1)
```
Comment (ken2403) [inline pyg_material/model/wfnet.py:201]: move a0 to basis func

File: pyg_material/model/wfnet.py [hunk 5]
```diff
@@ -307,6 +308,7 @@ def forward(self, x: Tensor, edge_index: Tensor, rbfs: Tensor, coeffs: Tensor) -
 
         # get rbf values
         rbfs = torch.einsum("ed,edh->eh", rbfs, coeffs)
+        rbfs = rbfs / (torch.sqrt(torch.sum(rbfs**2, dim=1, keepdim=True)) + 1e-12)
```
Comment (ken2403) [inline pyg_material/model/wfnet.py:311]: **important**
add normalization in convolution layer

File: pyg_material/model/wfnet.py [hunk 6]
```diff
@@ -318,21 +320,25 @@ def __init__(
...
         self.hidden_dim = hidden_dim
         self.out_dim = out_dim
+        self.aggr = aggr
 
         self.out_lin = nn.Sequential(
             activation,
             Dense(hidden_dim, hidden_dim, True, weight_init),
             activation,
-            Dense(hidden_dim, out_dim, False, weight_init),
+            Dense(hidden_dim, hidden_dim // 2, True, weight_init),
```
Comment (ken2403) [inline pyg_material/model/wfnet.py:334]: add output layer

File: pyg_material/model/wfnet.py [hunk 7]
```diff
@@ -150,15 +160,17 @@ def r_nl(r):
 
 
 class RadialWaveBasis(nn.Module):
-    n_radial: int = 15
+    n_radial: int = ELEC_DICT.size(-1)
```
Comment (ken2403) [inline pyg_material/model/wfnet.py:163]: **important**
remove standarization method and add radius param
